static/js/components/Select_test.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React from "react"
+import { assert } from "chai"
+import sinon from "sinon"
+import { shallow } from "enzyme/build"
+
+import Select from "./Select"
+
+describe("Select", function() {
+  let sandbox,
+    options,
+    isMulti,
+    closeMenuOnSelect,
+    openMenuOnClick,
+    placeholder,
+    menuPlacement,
+    className,
+    field,
+    form
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox()
+    closeMenuOnSelect = false
+    openMenuOnClick = false
+    isMulti = false
+    placeholder = "Select something..."
+    menuPlacement = "bottom"
+    className = "class"
+    field = { name: "TestName" }
+    ;(form = {}),
```

Comment:
there's a weird formatting thing here (prettier artifact?)

can change to just `form = {}` and `options = [ ... ]`